mock/resource/__init__.py

```diff
@@ -16,91 +16,126 @@
 from definitions import ROOT_DIR
 
 
-# TODO: Should this be a Singleton?
+@final
```

Comment:
Did not know about `final`. Thanks!

Reply:
I knew about `Final` in declaring constants, but not in classes and methods until just a few days ago.

Reply:
(as a decorator)